dags/hivemind_etl_helpers/tests/integration/test_github_etl_fetch_raw_issues.py

```diff
@@ -2,24 +2,27 @@
 from unittest import TestCase
 
 from github.neo4j_storage.neo4j_connection import Neo4jConnection
-from hivemind_etl_helpers.src.db.github.extract.issues import fetch_raw_issues
+from hivemind_etl_helpers.src.db.github.extract import GithubExtraction
 
 
 class TestGithubETLFetchRawIssues(TestCase):
     def setUp(self) -> None:
+        self.extractor = GithubExtraction()
         neo4j_connection = Neo4jConnection()
         self.neo4j_driver = neo4j_connection.connect_neo4j()
         with self.neo4j_driver.session() as session:
             session.execute_write(lambda tx: tx.run("MATCH (n) DETACH DELETE (n)"))
 
     def test_get_empty_results_no_from_date(self):
         repository_ids = [123, 124]
-        issues = fetch_raw_issues(repository_id=repository_ids, from_date=None)
+        issues = self.extractor._GithubIssueExtraction__fetch_raw_issues(
+            repository_id=repository_ids, from_date=None
+        )
         self.assertEqual(issues, [])
 
     def test_get_empty_results(self):
         repository_ids = [123, 124]
-        issues = fetch_raw_issues(
+        issues = self.extractor._GithubIssueExtraction__fetch_raw_issues(
```

Comment:
Please consider updating to use the `_fetch_raw_issues` directly and not via `GitHubIssueExtraction`